virtualizarr/tests/test_writers/test_icechunk.py

```diff
@@ -35,21 +42,19 @@ def icechunk_storage(tmp_path: Path) -> "Storage":
 
 @pytest.fixture(scope="function")
 def icechunk_repo(icechunk_storage: "Storage", tmp_path: Path) -> "Repository":
-    config = icechunk.RepositoryConfig.default()
 
-    url_prefixes = ["file:///private/var/folders/70", "file:///tmp/"]
```

Comment:
I don't know what this `/70` bit was, but that's not the correct path on my machine.